### PR TITLE
Changes for commissioning 2018

### PR DESCRIPTION
**Pull request by rauser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,19 +13,18 @@ git cms-init
 git remote add btv-cmssw https://github.com/cms-btv-pog/cmssw.git
 
 #Add DeepFlavour and its dependencies
-git cms-merge-topic capalmer85:btagSFupdatesForTTbar
-git cms-merge-topic cms-btv-pog:DeepFlavourUpdates-from-CMSSW_9_4_4
+git cms-merge-topic cms-btv-pog:DeepFlavourNewTraining-from-CMSSW_10_1_2_patch2
 
-git clone -b 9_4_X_v1.06 --depth 1 https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements.git RecoBTag/PerformanceMeasurements
+git clone -b https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements.git RecoBTag/PerformanceMeasurements
```
Comment (mverzett) [inline README.md]: This should be 
git clone -b 10_1_X --depth 1 blah blah
-b clones and checks out the 10_1_X branch. depth 1 slims the commit history to be manageable for the end user, you should not do it :)

File: python/defaults/Commissioning18.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+common = {
+	'eras' : ['Run2_2018'],
+	'miniAOD' : True,
+	'storeDeepFlavourTagVariables' : True,
+}
+
+mc = {
+        #'inputFiles' : ['/store/relval/CMSSW_10_1_0_pre2/RelValTTbar_13/MINIAODSIM/100X_upgrade2018_realistic_v11-v1/20000/14AC524C-981E-E811-B6D1-0CC47A4D7632.root'],
+        'inputFiles' : ['/store/mc/RunIISpring18MiniAOD/TTToHadronic_TuneCP5_13TeV-powheg-pythia8/MINIAODSIM/100X_upgrade2018_realistic_v10-v1/30000/008B1124-0B31-E811-BD14-0025904C7A5A.root'],
```
Comment (mverzett) [inline python/defaults/Commissioning18.py]: This should be some 10.1.X relVal file, have a look at https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatAlgos/python/patInputFiles_cff.py

**Comment by mverzett** [comment]
@rauser just to confirm, in this case the negative tagInfos will be defaulted to -10 rather than not being stored at all, right?

Apart from the other two comments everything looks good though :)

File: python/defaults/Commissioning18.py (hunk 2 above)
Comment (rauser) [reply python/defaults/Commissioning18.py]: Is the file which is currently commented ok? This was previously pointed out by Ivan. In the link I couldn't find another 10.1.X relVal file

Comment (mverzett) [reply python/defaults/Commissioning18.py]: yeah, that works. please update and just push the new commit to the same branch. Git will do its magic and append it to this PR

**Comment by rauser** [comment]
The negative tagInfos will be filled with a default value which is the result of the computations using the default value of -10 for all inputs. The resulting values are: 
Jet_DeepFlavourBDiscN: -3000
Jet_DeepFlavourCvsLDiscN: 0.3333
Jet_DeepFlavourCvsBDiscN: 0.25
Is this behaviour accectable or should all of them be filled with a more obvious default value?

**Comment by carolinecollard** [comment]
Hi all,

Would it be possible to get default value for Jet_DeepFlavourCvsLDiscN and Jet_DeepFlavourCvsBDiscN which are not in the middle of the distribution? At present they could translate into spikes between two potential WP selections (if for example we take the same WP cuts as for the DeepCSV charm-tagger, then we might be in troubles).

By the way, are these default values the same as for the standard taggers (without the N, I mean) ? 

Cheers,
Caroline

**Comment by rauser** [comment]
Hey Caroline,

I just made a new pull request, updating the default values for both the standard and negative taggers to -1000. See https://github.com/cms-btv-pog/RecoBTag-PerformanceMeasurements/pull/87

In the previous version, the standard taggers had the same default value computed as the negative taggers.

Cheers, Johann

**Pull request merged**